docs/apache-airflow/best-practices.rst

```diff
@@ -228,11 +228,36 @@ Do not hard code values inside the DAG and then change them manually according t
 
 You can use environment variables to parameterize the DAG.
 
-.. code-block::
+.. code-block:: python
+
+   import os
+
+   dest = os.environ.get(
+      "MY_DAG_DEST_PATH",
+      "s3://default-target/path/"
+   )
+
+Mocking variables and connections
+=================================
+
```

Comment:
I think it would be great to add a little context here. In which situations you would like to do that? I am not 100% sure here if we are talking about "user facing' mocking or "contributors" mocking. If this is user-facing, then it would be great to explain when mocking might be useful. If this is "contributors-facing" then it should be moved to CONTRIBUTORS.

Reply:
I think this is useful for the end user. I've seen similar questions on Slack many times, including once yesterday. See: https://apache-airflow.slack.com/archives/CSS36QQS1/p1609874991010700

I have added some context in which cases, this can be helpful.